scheds/include/scx/common.bpf.h

```diff
@@ -706,6 +726,25 @@ static inline u32 log2_u64(u64 v)
                 return log2_u32(v) + 1;
 }
 
+/*
+ * sqrt_u64 - Calculate the square root of value @x using Newton's method.
+ */
+static inline u64 __sqrt_u64(u64 x)
```

Comment:
We could also copy https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/lib/math/int_sqrt.c (we should be able to reuse it in BPF with some minimal changes).

Not for this PR of course, as a future improvement.